### ISSUE TITLE
Openshift3.9  - The volume mount operation fails intermittently with an error message

### ISSUE THREAD

**Issue by dockerciuser** [hpe-storage/python-hpedockerplugin#234]
The console output for this issue is - 

Events:
  Type     Reason                 Age               From                             Message
  ----     ------                 ----              ----                             -------
  Normal   SuccessfulMountVolume  6m                kubelet, openshift-master-node1  MountVolume.SetUp succeeded for volume "default-token-c8d2x"
  Warning  FailedMount            2m (x2 over 4m)   kubelet, openshift-master-node1  Unable to mount volumes for pod "pod-test8_default(8049220a-940a-11e8-b732-c4346bc12918)": timeout expired waiting for volumes to attach/mount for pod "default"/"pod-test8". list of unattached/unmounted volumes=[export]
  Warning  FailedMount            41s (x6 over 5m)  kubelet, openshift-master-node1  MountVolume.SetUp failed for volume "sc-test8-82681d06-9409-11e8-b732-c4346bc12918" : mount command failed, status: Failure, reason: Post http://unix/VolumeDriver.Mount: http: ContentLength=101 with Body length 0
        

Also find the attached logs for more details.
[messages_log_obs_2.txt](https://github.com/hpe-storage/python-hpedockerplugin/files/2269097/messages_log_obs_2.txt)

**Comment by farhan7500** [comment]
@dockerciuser: Please could you add more information here. Basically I am looking for 
- How to reproduce this issue
- Container logs, dory logs and messages logs

I could reproduce this on my setup yesterday and I want to know if the two issues are the same. I mean, just because I got the same error exception does not necessarily mean I reproduced it exactly. Here are the details:

One of the two iscsi targets on the array was not logging in:
`2018-10-22 08:38:47.012 18 WARNING os_brick.initiator.connectors.iscsi [-] Failed to login iSCSI target iqn.2000-05.com.3pardata:20620002ac0193b9 on portal 15.212.193.101:3260 (exit code 8)`

One of the targets failed to connect:
`
[root@localhost ~]# iscsiadm -m node --targetname "iqn.2000-05.com.3pardata:20610002ac0193b9" --portal "15.212.192.112:3260" --login
Logging in to [iface: default, target: iqn.2000-05.com.3pardata:20610002ac0193b9, portal: 15.212.192.112,3260] (multiple)
Login to [iface: default, target: iqn.2000-05.com.3pardata:20610002ac0193b9, portal: 15.212.192.112,3260] successful.  
The port at 15.212.192.112 connects successfully but the node at 15.212.193.101 fails to connect
`
Is this what you also saw. If not please could you send over more information with the applicable logs for me to move ahead.

**Comment by dockerciuser** [comment]
@farhan7500 The setup details of Openshift 3.9 in FTC are  - 
10.50.9.15 master
10.50.9.16 worker1
10.50.9.17 worker2

There are specific steps to reproduce this issue. I observed this intermittently, in POD creation.

**Comment by farhan7500** [comment]
Should I access these through a jump machine? What are the credentials of jump machine and the nodes?

**Comment by farhan7500** [comment]
Got my answer from William for:
> Should I access these through a jump machine? What are the credentials of jump machine and the nodes?

Which plugin container should I use to reproduce the issue?

**Comment by farhan7500** [comment]
Created POD multiple times as this is supposed to be intermittent, but failed to reproduce this issue. I have created a script that will repeatedly create the POD, and destroy it once it successful to reproduce this.

However, faced another issue where the PVC went into lost state. I have described this and have attached the logs at https://github.com/hpe-storage/python-hpedockerplugin/issues/364#issuecomment-433383487
